app/controllers/green_lanes/applicable_exemptions_controller.rb

```diff
@@ -135,6 +149,7 @@ def handle_next_page(next_page)
         commodity_code: params[:commodity_code],
         country_of_origin: params[:country_of_origin],
         moving_date: params[:moving_date],
+        category: params[:category],
```

Comment:
Does this duplicate the category above?

Reply:
yes maybe this might have been a bad rebase 👍